src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/simplifybooleanreturn/InputSimplifyBooleanReturnWithYield.java

```diff
@@ -0,0 +1,46 @@
+/*
+SimplifyBooleanReturn
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.coding.simplifybooleanreturn;
+
+public class InputSimplifyBooleanReturnWithYield {
```

Comment:
Please add inputs with case labels (`:`) and also equivalent switches (where possible) with `return`.

Reply:
done